Source/DafnyCore/Verifier/BoogieGenerator.Functions.Wellformedness.cs

```diff
@@ -392,12 +392,18 @@ private List<Variable> GetWellformednessProcedureOutParameters(Function f, Expre
 
       if (generator.options.Get(CommonOptionBag.CheckInvariants)) {
         // NB(somayyas): need to frame open here in case the function calls any invariant
-        etran.OpenFormal(f.Origin, out var openBoogie, out var openDafny);
-        var openIsEmpty = new BinaryExpr(f.Origin, BinaryExpr.ResolvedOpcode.SetEq, openDafny,
+        etran.OpenFormal(f.Origin, out var openBoogie, out var openExprDafny);
+        var openIsEmpty = new BinaryExpr(f.Origin, BinaryExpr.ResolvedOpcode.SetEq, openExprDafny,
           new SetDisplayExpr(f.Origin, true, [])
             { Type = generator.program.SystemModuleManager.NonNullObjectSetType(f.Origin) });
         requires.Add(generator.Requires(f.Origin, true, openIsEmpty, etran.TrExpr(openIsEmpty),
           null, null, "open set frame condition"));
+        /*if (!f.IsStatic) {
```

Comment:
What I would do it maintain a branch that has the code you want to use later, but in this PR remove all that code. You can have a single commit that removes it, so it'll be easy to revert later.

We don't know what the future code will look like, so leaving in comments such as these risks those being forgotten and staying in for a long time.